code/parachain/runtime/picasso/src/lib.rs

```diff
@@ -1230,3 +1464,27 @@ cumulus_pallet_parachain_system::register_validate_block!(
 	BlockExecutor = cumulus_pallet_aura_ext::BlockExecutor::<Runtime, Executive>,
 	CheckInherents = CheckInherents,
 );
+
+impl pallet_ibc::Config for Runtime {
+	type TimeProvider = Timestamp;
+	type RuntimeEvent = RuntimeEvent;
+	type NativeCurrency = Balances;
+	type Balance = Balance;
+	type AssetId = CurrencyId;
+	type NativeAssetId = NativeAssetId;
+	type IbcDenomToAssetIdConversion = IbcDenomToAssetIdConversion;
+	const PALLET_PREFIX: &'static [u8] = b"ibc/";
+	const LIGHT_CLIENT_PROTOCOL: pallet_ibc::LightClientProtocol =
+		pallet_ibc::LightClientProtocol::Grandpa;
+	type AccountIdConversion = ibc_primitives::IbcAccount<AccountId>;
+	type Fungibles = Assets;
+	type ExpectedBlockTime = ExpectedBlockTime;
+	type Router = Router;
+	type MinimumConnectionDelay = MinimumConnectionDelay;
+	type ParaId = parachain_info::Pallet<Runtime>;
+	type RelayChain = RelayChainId;
+	type WeightInfo = ();
+	type AdminOrigin = EnsureRoot<AccountId>;
```

Comment:
This should be RootOrHalfCouncil

Reply:
do you mean
`type AdminOrigin = EnsureRootOrHalfNativeCouncil;`